### PR TITLE
commented out sponsore placement component in the homepageTopper

### PR DESCRIPTION
**Pull request by bonomite**

### CODE REVIEW

File: components/GothamistHomepageTopper.vue [hunk 1]
```diff
@@ -12,12 +12,15 @@ const props = defineProps<{
...
 const latestArticles = computed(() => {
-  return props.articles.slice(1)
-    // remove the manually curated featured article
-    // from the list of latest articles so it
-    // doesn't show up twice in this module
-    .filter(article => article.uuid !== featuredArticle.value.uuid)
-    .slice(0,4)
+  return (
+    props.articles
+      .slice(1)
```
Comment (walsh9) [inline components/GothamistHomepageTopper.vue:17]: We're already removing the featuredArticle from the latest articles list with the .filter below. What is this trying to do?

File: components/GothamistHomepageTopper.vue [hunk 2]
```diff
@@ -54,7 +57,7 @@ const latestArticles = computed(() => {
     </div>
     <div class="col-12 xl:col-4 flex flex-column justify-content-end">
       <hr class="black mb-3 xl:hidden" />
-      <SponsorPlacement />
+      <!-- <SponsorPlacement /> -->
```
Comment (walsh9) [inline components/GothamistHomepageTopper.vue:60]: Why not just delete this?

**Review by walsh9 — CHANGES_REQUESTED** [review]

**Review by kimlarocca — APPROVED** [review]

File: components/GothamistHomepageTopper.vue (hunk 2 above)
Comment (walsh9) [reply components/GothamistHomepageTopper.vue:60]: Actually, we can probably delete the component too.

Comment (walsh9) [reply components/GothamistHomepageTopper.vue:60]: And the logo images too.

File: components/GothamistHomepageTopper.vue (hunk 1 above)
Comment (bonomite) [reply components/GothamistHomepageTopper.vue:17]: ah, this is just formatting...

File: components/GothamistHomepageTopper.vue (hunk 2 above)
Comment (bonomite) [reply components/GothamistHomepageTopper.vue:60]: ok sounds good

File: components/GothamistHomepageTopper.vue (hunk 1 above)
Comment (walsh9) [reply components/GothamistHomepageTopper.vue:17]: Ah i see, hmm maybe i wrote that one and meant to do a slice(0) to make a copy, or maybe i didn't see it there when I wrote the bottom part.

**Review by walsh9 — APPROVED** [review]

File: components/GothamistHomepageTopper.vue (hunk 1 above)
Comment (walsh9) [reply components/GothamistHomepageTopper.vue:17]: I mean it's redundant and should be cleaned up, but it's not causing any problems, and not really in scope for this ticket. I'm sure we'll touch loading articles or implementing other types of toppers or something like that at some point and can revisit it then.

**Pull request merged**